core-httpclient-impl/src/main/java/com/optimizely/ab/config/HttpProjectConfigManager.java

```diff
@@ -73,11 +74,13 @@ private HttpProjectConfigManager(long period,
                                      String datafileAccessToken,
                                      long blockingTimeoutPeriod,
                                      TimeUnit blockingTimeoutUnit,
-                                     NotificationCenter notificationCenter) {
+                                     NotificationCenter notificationCenter,
+                                     String sdkKey) {
```

Comment:
Can we also avoid this change as well if we can get sdkKey via ProjectConfig?

Reply:
Same as above we need to have sdkKey pass in here, to make sure that the when we call getSDKKey on projectConfigManager then we won't get it null in case if it is http config Manager.